hubblestack/utils/signing.py

```diff
@@ -124,10 +167,14 @@ def split_certs(fh):
             if line.startswith('----'):
                 ret = ret.encode()
                 try:
+                    log_level = log.debug
                     yield ossl.load_certificate(ossl.FILETYPE_PEM, ret)
-                except Exception as exception_object:
-                    log.debug('decoding item as certificate failed: %s; trying as PEM encoded private key',
-                        exception_object)
+                except Exception as E:
+                    status = STATUS.UNKNOWN
+                    if check_verif_timestamp(fh) == True:
+                        log_level = log.error
+                    msg = '{}: | file: "{}" | cert decoding status: {} | attempting as PEM encoded private key'
+                    log_level(msg.format(short_fname, fh.name, status, digest, code, depth, message))
```

Comment:
Avoid using `msg.format`; log.debug already issues a format, but using the older style (for various reasons). Please change to `log_level('%s: | file: "%s" | cert decoding status: %s | attempting as PEM encoded private key', short_fname, fh.name, ...)`

Reply:
Got it, will format using the older style